app/views/posts/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Blog App</h1>
+
+<div>
+  <%= form_with model: @post, class:'form', url: user_posts_path(@user) do |form| %>
+    <%= form.label :title, class: 'post_label' %>
+    <%= form.text_field :title, required: true, class:'post_title' %>
+    <%= form.label :text, class: 'post_label' %>
+    <%= form.text_area :text, required: true, class:'post_text' %>
+    <%= form.submit "Post", class: 'post_btn' %>
+  <% end %>
+</div>
```

Comment:
- [ ] As mentioned by the previous reviewer, there should a form for creating posts on the user list of posts. At this moment, that for does not exist now. Take a look at below video record.

### Your App Behavior.

[no-form.webm](https://github.com/uno36/blog-app-rails/assets/26597432/6b50d367-2980-41d0-9257-fb9de53e46a9)